1_assignment/alphbet_frequency.js

```diff
@@ -1,13 +1,18 @@
 
-string = "u@#d$fkj$#!kds"
 
-function calculateFrequency(string) {
+//Sample String .
+var string = "u@#d$fkj$#!kds";
+
+
+
+// Solution 1 : Using String Functions.
+function calculateFrequencyUsingString(string) {
   // Write your code here
   var i=0;
   frequency=new Array(26).fill(0);
```

Comment:
var keyword missing and formatting still incorrect. Global variables should be as limited as possible and only used where utmost necessary.